docs/cloud/insights/events-feed.md

```diff
@@ -0,0 +1,59 @@
+<!--
+title: "Events feed"
+sidebar_label: "Events feed"
+custom_edit_url: "https://github.com/netdata/netdata/blob/master/docs/cloud/insights/events-feed.md"
+sidebar_position: "2800"
+learn_status: "Published"
+learn_topic_type: "Concepts"
+learn_rel_path: "Concepts"
+learn_docs_purpose: "Present the Netdata Events feed."
+-->
+
+Netdata Cloud provides the Events feed which is a powerful feature that tracks events that happen on your infrastructure, or in your Space. The feed lets you investigate events that occurred in the past, which is obviously invaluable for troubleshooting. Common use cases are ones like when a node goes offline, and you want to understand what events happened before that. A detailed event history can also assist in attributing sudden pattern changes in a time series to specific changes in your environment.
+
+#### What are the available events?
+
+At a high-level view, these are the domains from which the Events feed will provide visibility into.
+
+> ⚠️ Based on your space's plan different allowances are defined to query past data.
+
+| **Domains of events** | **Community** | **Pro** | **Business** |
+| :-- | :-- | :-- | :-- |
+| **Auditing events** - COMING SOON<br/>Events related to actions done on your Space, e.g. invite user, change user role or change plan.| 4 hours | 7 days | 90 days |
+| **[Topology events](#topology-events)**<br/>Node state transition events, e.g. live or offline.| 4 hours | 7 days | 14 days |
+| **[Alert events](#alert-events)**<br/>Alert state transition events, can be seen as an alert history log.| 4 hours | 7 days | 90 days |
+
+##### Topology events
+
+| **Event name** | **Description** |  **Example** |
+| :-- | :-- | :-- |
+| Node Become Live | The node is collecting and streaming metrics to Cloud.| Node `netdata-k8s-state-xyz` is on live **state** |
+| Node Become Stale | The node is offline and not streaming metrics to Cloud. It can show historical data from a parent node. | Node `ip-xyz.ec2.internal` is on **stale** state |
+| Node Become Offline | The node is offline, not streaming metrics to Cloud and not available in any parent node.| Node `ip-xyz.ec2.internal` is on **offline** state |
+| Node Created | The node is created but it is still Unseen on Cloud, didn't establish a successful connection yet.| Node `ip-xyz.ec2.internal` is on **unseen** state |
+| Node Removed | The node was removed from the Space through the `Delete` action, if it becomes Live again it will be automatically added. | Node `ip-xyz.ec2.internal` was **deleted(soft)** |
+| Node Restored | The node is restored, if node becomes Live after a remove action it is re-added to the Space. | Node `ip-xyz.ec2.internal` was **restored** |
```

Comment:
In both the above, the comma-separated parts of the sentences don't work for me. Also nodes currently are only removed by the user (`event.reason` is `user-request`), but this may change.

| Action | Description | Example |
|---|---|---|
| Node Removed |The node was removed from the Space, for example by using the `Delete` action on the node. This is a soft delete in that the node gets marked as deleted, but retains the association with this space. If it becomes live again, it will be restored (see `Node Restored` below) and reappear in this space as before. | Node `ip-xyz.ec2.internal` was **deleted (soft)** |
| Node Restored | The node was restored. See `Node Removed above. |Node `ip-xyz.ec2.internal` was **restored** |